packages/react/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,276 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+import { handleRef, Ref } from '@stardust-ui/react-component-ref'
+import cx from 'classnames'
+
+import {
+  applyAccessibilityKeyHandlers,
+  AutoControlledComponent,
+  ChildrenComponentProps,
+  commonPropTypes,
+  isFromKeyboard,
+  partitionHTMLProps,
+  UIComponentProps,
+  RenderResultConfig,
+  setWhatInputSource,
+} from '../../lib'
+import {
+  ComponentEventHandler,
+  ShorthandValue,
+  WithAsProp,
+  withSafeTypeForAs,
+  Omit,
+} from '../../types'
+import { Accessibility } from '../../lib/accessibility/types'
+import { sliderBehavior } from '../../lib/accessibility'
+import { SupportedIntrinsicInputProps } from '../../lib/htmlPropsUtils'
+import Box from '../Box/Box'
+
+const defaultPropValues = {
```

Comment:
Why not inlining these to the defaultProps? It's hard to follow what is actually in the default props

Reply:
done, I was using them outside the component and thought it's more obvious, but managed to find a way, thanks!